lalrpop/src/build/mod.rs

```diff
@@ -186,16 +186,17 @@ fn lalrpop_files<P: AsRef<Path>>(root_dir: P) -> io::Result<Vec<PathBuf>> {
             result.extend(lalrpop_files(&path)?);
         }
 
-        let is_symlink_file = || -> io::Result<bool> {
+        let is_symlink_file = || -> bool {
```

Comment:
Nitpick: I wonder if this small lambda could reflect slightly better the new semantics, something like `is_valid_symlink_file`, or a better word? It's of course not very important, you can dismiss if you feel like it

Reply:
Sure, seems reasonable